### ISSUE TITLE
Conv layers overrides data_format passed to the layer with keras config image_data_format

### ISSUE THREAD

**Issue by sandeep-krishnamurthy** [awslabs/keras-apache-mxnet#121]
Convolution layers with MXNet backend, uses keras config image_data_format for preparing the kernels. It ignores data_format passed to the layer explicitly.

This is a rare case from usage perspective where users use data_format parameter of the layer. Majority of the users use ~/.keras/keras.json config file to set the image_data_format.

This needs to be fixed. Few test cases in convolutional layer tests needs to be enabled after the fix.

**Comment by roywei** [comment]
test_conv_1d with channels_first test is disabled:
```
# Explicitly setting data_format has issues with MXNet backend in Conv operators.
    if K.backend() != 'mxnet':
        # Test channels_first
        layer_test(convolutional.Conv1D,
                   kwargs={'filters': filters,
                           'kernel_size': kernel_size,
                           'data_format': 'channels_first'},
                   input_shape=(batch_size, input_dim, steps))
```

might be related to #122  as imdb_cnn use Conv1D.

**Comment by roywei** [comment]
fixed with #143 , closing
